tasks/main.yml

```diff
@@ -15,9 +15,18 @@
 - name: Add Docker repository.
   apt_repository: repo='deb https://get.docker.io/ubuntu docker main' state=present
 
-- name: Install packages
-  apt: name="{{ item }}" update_cache=yes
-  with_items: packages
+- name: Install support packages
+  apt: name="{{ item }}"
+  with_items:
+    - wget
+
+- name: Install docker package
+  apt: name="{{ item }}{% if docker_version is defined %}={{ docker_version }}{% endif %}"
+  with_items:
+    - lxc-docker
+
+- name: Do not use sysv to start docker daemon
```

Comment:
Looks like it was fixed in 1.9.2: ansible/ansible-modules-core@cbbb20b6bf0d82fdc92f064c9e4d18c04f203da6

So I think it'd be sufficient to bump our `requirements.txt` in alphagov/tsuru-ansible